config/install/field.storage.media.field_image_storage.yml

```diff
@@ -0,0 +1,23 @@
+uuid: bb6562e5-1092-4f52-81d2-fe582c311020
+langcode: en
+status: true
+dependencies:
+  module:
+    - file
+    - media_entity
+id: media.field_image_storage
+field_name: field_image_storage
+entity_type: media
+type: file
```

Comment:
This needs to be an 'Image' field.  When you go to create the field, it's in the drop-down for field type.  It's pretty much the same as file, except it has some niceties for alt text and displaying the image.